admin/src/V3DeprecationWarning.js

```diff
@@ -7,7 +7,8 @@ import styles from './V3DeprecationWarning.module.css'
 import classnames from 'classnames'
 import Alert from './Alert'
 import { __, sprintf } from '@wordpress/i18n'
-import { createInterpolateElement } from '@wordpress/element'
+import { __experimentalCreateInterpolateElement, createInterpolateElement as stableCreateInterpolateElement } from '@wordpress/element'
+const createInterpolateElement = stableCreateInterpolateElement || __experimentalCreateInterpolateElement
```

Comment:
Might be nice to not repeat this everytime. Could make a wrapper module that does this and then exports the correct one.

```
import { createInterpolateElement } from './compat'
```

...or some ilk

Reply:
good idea. done.